client/src/mockData/questionsMock.js

```diff
@@ -1,37 +1,208 @@
+          'answerer_name': 'metslover',
+          'helpfulness': 5,
+          'photos': ['https://images.unsplash.com/photo-1530519729491-aea5b51d1ee1?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1651&q=80', 'https://images.unsplash.com/photo-1511127088257-53ccfcc769fa?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80', 'https://images.unsplash.com/photo-1500603720222-eb7a1f997356?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1653&q=80']
+        },
+        '1444525': {
+          'id': 1444525,
+          'body': 'Its the best! Seriously magic fabric',
+          'date': '2018-01-04T00:00:00.000Z',
+          'answerer_name': 'metslover',
+          'helpfulness': 7,
+          'photos': []
+        },
+        '1444526': {
+          'id': 1444526,
+          'body': 'DONT BUY IT! It\'s bad for the environment',
+          'date': '2018-01-04T00:00:00.000Z',
+          'answerer_name': 'metslover',
+          'helpfulness': 8,
+          'photos': []
+        },
+        '1444575': {
+          'id': 1444575,
+          'body': 'Suede',
+          'date': '2018-11-04T00:00:00.000Z',
+          'answerer_name': 'metslover',
+          'helpfulness': 7,
+          'photos': []
+        },
+        '1444613': {
+          'id': 1444613,
+          'body': 'Supposedly suede, but I think its synthetic',
+          'date': '2018-12-04T00:00:00.000Z',
+          'answerer_name': 'metslover',
+          'helpfulness': 3,
+          'photos': []
+        }
+      }
+    },
+    {
+      'question_id': 212330,
+      'question_body': 'Why is abbreviation such a long word?',
+      'question_date': '2021-06-06T00:00:00.000Z',
+      'asker_name': 'jack handy',
+      'question_helpfulness': 0,
+      'reported': false,
+      'answers': {}
+    },
+    {
+      'question_id': 212329,
+      'question_body': 'Why is abbreviation such a long word?',
+      'question_date': '2021-06-06T00:00:00.000Z',
+      'asker_name': 'jack handy',
+      'question_helpfulness': 0,
+      'reported': false,
+      'answers': {}
+    },
+    {
+      'question_id': 212328,
+      'question_body': 'Why is abbreviation such a long word?',
+      'question_date': '2021-06-06T00:00:00.000Z',
+      'asker_name': 'jack handy',
+      'question_helpfulness': 0,
+      'reported': false,
+      'answers': {}
+    },
+    {
+      'question_id': 212327,
+      'question_body': 'Why is abbreviation such a long word?',
+      'question_date': '2021-06-06T00:00:00.000Z',
+      'asker_name': 'jack handy',
+      'question_helpfulness': 0,
+      'reported': false,
+      'answers': {}
+    },
+    {
+      'question_id': 212326,
+      'question_body': 'Why is abbreviation such a long word?',
+      'question_date': '2021-06-06T00:00:00.000Z',
+      'asker_name': 'jack handy',
+      'question_helpfulness': 0,
+      'reported': false,
+      'answers': {}
+    }
   ]
 };
 
 const answersMock = {
-  "question": "1",
```

Comment:
In all actuality we should have either put the mock data into json files, or converted to appropriately styled objects/arrays. I just didn't want to make everyone create json parsing functions to get the client rendering right away.

Reply:
Agree